content/advanced-training/map/house.md

```diff
@@ -34,34 +34,33 @@ type = "map"
 
 The setting for this map is a United States suburban neighborhood house. With four spawn locations, two floors and a basement, the house offers good options for approach and entry.
 
-## Drone Entry Points
-
-The house has 5 possible entry points for your remote controlled surveillance drone.
-
-1. Basement - Northwest corner underneath the Storage Room door
-1. Basement - Eastern side through the debris of the Garage door (between the two overhead doors)
-1. 1st Floor - North side of the home under the barricade on the main entryway
-1. 1st Floor - South side of the home, over the furniture blocking the back entryway
-1. 1st Floor - Southwest corner of the home, under a barricade to the hall between the West stairs and the kitchen
-1. 1st Floor - Northeast corner of house entering under a barricade into the office
-
 ## Spawn Locations
 
 ### Construction Site
 
-The construction site spawn positions you North of the house. The main entrance to the house on the 1st floor is the closest drone entry point.
+The construction site spawn positions you North of the house.
+
+* **Drone entry:** The main entrance to the house on the [1st floor]({{< ref "#1st-floor-entry">}}) is the closest entry point for your drone.
+* **Camera:** Your approach to the house is monitored by the [front side camera]({{< ref "#front-camera">}}) located to the right near the property perimeter.
 
 ### APC
 
-The APC (Armored Personnel Carrier) drops you outside the house to the South side. The closest drone entry point is through the barricaded doorway on the Southwest corner of the house on the 1st floor.
+The APC (Armored Personnel Carrier) drops you outside the house to the South side.
+
+* **Drone entry:** The barricaded doorway near the Southwest corner of the house on the [1st floor]({{< ref "#1st-floor-entry">}}) is the closest entry point for your drone.
+* **Camera:** Your approach to the house is monitored by the [rear camera]({{<ref "#rear-camera">}}) located in the Southeast side of the house, along the perimeter of the property.
```

Comment:
Can you provide directions to the camera based off of their approach to the house, e.g. to the right from where you spawn.